_projects/pmefr_project.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page_project
+title: Programming Model Extensions for Resilience
+date: 2016-02-10
+updated:
+navbar: Projects
+footer: true
+project_url:
+status: starting
+topics:
+  resilience
```

Comment:
`topics` should be a list of topics, thus use one of the following syntaxes:
```
topics:
  - topic_id1
  - topic_id2
```
or
```
topics: [topic_id1, topic_id2]
```